qutebrowser/browser/pdfjs.py

```diff
@@ -44,6 +44,63 @@ def __init__(self, path):
         super().__init__(message)
 
 
+def _run_js(browsertab, code):
+    # Execute javascript code globally.
+    if objects.backend == usertypes.Backend.QtWebEngine:
+        from PyQt5.QtWebEngineWidgets import QWebEngineScript
+        browsertab.run_js_async(code, world=QWebEngineScript.MainWorld)
+    else:
+        browsertab.run_js_async(code)
+
+
+def is_pdfjs_page(browsertab):
+    """Check whether the given browsertab displays a pdfjs site.
+
+    Args:
+        browsertab: The qutebrowser.browser.browsertab.AbstractTab instance.
+
+    Return:
+        True if the view displays pdfjs, False otherwise.
+    """
+    if objects.backend == usertypes.Backend.QtWebEngine:
+        url = browsertab.url().url()
+        return url.startswith('qute://pdfjs/web/viewer.html')
+    else:
+        # TODO implement this
+        return False
+
+
+def zoom_in(browsertab, count=1):
+    """Increase the zoom level for the current tab, using pdfjs zoom feature.
+
+    Args:
+        browsertab: The qutebrowser.browser.browsertab.AbstractTab instance.
+        count: How many steps to zoom in.
+    """
+    _run_js(browsertab, 'PDFViewerApplication.zoomIn({})'.format(count))
```

Comment:
Like you said, this probably only works in newer PDF.js versions. I'd like to keep supporting older versions if possible though (at least whatever e.g. Debian Jessie packages system-wide).

Reply:
Also, please use `utils.javascript.to_js()` everwhere you format a value into JS code.